server/routes/api.js

```diff
@@ -290,6 +290,20 @@ router.post('/setup', (req, res) => {
   })
 });
 
+router.post('/import/dashboards', (req, res) => {
```

Comment:
Done. as discussed, now /dashboards/id
and the content is sent as part of a PUT command